django/core/management/base.py

```diff
@@ -244,18 +244,20 @@ def get_version(self):
         """
         return django.get_version()
 
-    def create_parser(self, prog_name, subcommand):
+    def create_parser(self, prog_name, subcommand, **kwargs):
         """
         Create and return the ``ArgumentParser`` which will be used to
-        parse the arguments to this command.
+        parse the arguments to this command. Hi.
         """
-        parser = CommandParser(
-            prog='%s %s' % (os.path.basename(prog_name), subcommand),
-            description=self.help or None,
-            formatter_class=DjangoHelpFormatter,
-            missing_args_message=getattr(self, 'missing_args_message', None),
-            called_from_command_line=getattr(self, '_called_from_command_line', None),
-        )
```

Comment:
What about just adding `**kwargs` here? It should be the same but without the need for the creation of an intermediate dictionary.

Reply:
Yes, you're right. Same result, much simpler.